### PR TITLE
Create store

### PR DESCRIPTION
**Pull request by kushalShukla-web**
## Description

as createStore is no longer used by developer today days. i import it using "import { legacy_createStore as createStore} from 'redux'".
## Motivation and Context

it solves createStore problem . you need to import it using legacy_createStore .
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/AuthPage/ForgotPassword/index.js [hunk 1]
```diff
@@ -26,15 +26,25 @@ const ForgotPassword = ({
   const [error, setError] = useState("");
   const [success, setSuccess] = useState(false);
   const [email, setEmail] = useState("");
+  const [conform, conformemail] = useState(true);
```
Comment (ABHISHEK-PANDEY2) [inline src/components/AuthPage/ForgotPassword/index.js:29]: use better naming conventions for the states

File: src/components/AuthPage/ForgotPassword/index.js [hunk 2]
```diff
@@ -26,15 +26,25 @@ const ForgotPassword = ({
...
   const [open, setOpen] = React.useState(true);
-  const errorProps = useSelector(({ auth }) => auth.profile.error);
+  const errorProps = useSelector(({ auth }) => {
+  console.log(auth)
```
Comment (ABHISHEK-PANDEY2) [inline src/components/AuthPage/ForgotPassword/index.js:32]: remove this change

File: src/components/AuthPage/ForgotPassword/index.js [hunk 3]
```diff
@@ -26,15 +26,25 @@ const ForgotPassword = ({
...
+});
   const loadingProps = useSelector(({ auth }) => auth.profile.loading);
   const dispatch = useDispatch();
 
   useEffect(() => setError(errorProps), [errorProps]);
   useEffect(() => setLoading(loadingProps), [loadingProps]);
   useEffect(() => setOpen(true), [loadingProps]);
-
+  useEffect(()=>{
+    if(email.includes('@gmail.com') && /[0-9]/.test(email) && /\p{L}/u.test(email)){
```
Comment (ABHISHEK-PANDEY2) [inline src/components/AuthPage/ForgotPassword/index.js:42]: what about the emails containing "yahoo" or other such email services ?, I guess that would be a problem if we set it only for gmail

File: src/components/AuthPage/ForgotPassword/index.js [hunk 4]
```diff
@@ -26,15 +26,25 @@ const ForgotPassword = ({
...
+              conformemail(false);
```
Comment (ABHISHEK-PANDEY2) [inline src/components/AuthPage/ForgotPassword/index.js:43]: And also displaying an appropriate error message would be helpful

**Review by ABHISHEK-PANDEY2 — CHANGES_REQUESTED** [review]

File: src/components/AuthPage/ForgotPassword/index.js (hunk 3 above)
Comment (kushalShukla-web) [reply src/components/AuthPage/ForgotPassword/index.js:42]: but in codelabz website it only have gmail option.

Comment (Sarfraz-droid) [reply src/components/AuthPage/ForgotPassword/index.js:39]: Thanks @ABHISHEK-PANDEY2 for pointing out!

@kushalShukla-web the email - password sign in can have other emails too,
you can use something like `^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$` to verify the entire email at once!

**Comment by Sarfraz-droid** [comment]
Nice work!
@kushalShukla-web However, this branch also has merge conflicts at `src/components/AuthPage/ForgotPassword/index.js
` due to recent commits. can you go over through it? Thankyou

**Comment by kushalShukla-web** [comment]
Ya I will do it . Actually I have a problem with .y git

**Pull request closed without merging**